manifests/base/jupyter-tensorflow-notebook-imagestream.yaml

```diff
@@ -17,27 +17,27 @@ spec:
   tags:
     # N Version of the image
     - annotations:
-        opendatahub.io/notebook-software: '[{"name":"CUDA","version":"12.1"},{"name":"Python","version":"v3.9"},{"name":"TensorFlow","version":"2.15"}]'
-        opendatahub.io/notebook-python-dependencies: '[{"name":"TensorFlow","version":"2.15"},{"name":"Tensorboard","version":"2.15"},{"name":"Boto3","version":"1.34"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp","version":"2.5"},{"name":"Matplotlib","version":"3.8"},{"name":"Numpy","version":"1.26"},{"name":"Pandas","version":"2.2"},{"name":"Scikit-learn","version":"1.4"},{"name":"Scipy","version":"1.12"},{"name":"Odh-Elyra","version":"3.16"},{"name":"PyMongo","version":"4.6"},{"name":"Pyodbc","version":"5.1"}, {"name":"Codeflare-SDK","version":"0.19"}, {"name":"Sklearn-onnx","version":"1.16"}, {"name":"Psycopg","version":"3.1"}, {"name":"MySQL Connector/Python","version":"8.3"}]'
+        opendatahub.io/notebook-software: '[{"name":"CUDA","version":"12.4"},{"name":"Python","version":"v3.11"},{"name":"TensorFlow","version":"2.17"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"TensorFlow","version":"2.17"},{"name":"Tensorboard","version":"2.17"},{"name":"Boto3","version":"1.35"},{"name":"Kafka-Python","version":"2.2"},{"name":"Kfp","version":"2.5"},{"name":"Matplotlib","version":"3.9"},{"name":"Numpy","version":"1.26"},{"name":"Pandas","version":"2.2"},{"name":"Scikit-learn","version":"1.5"},{"name":"Scipy","version":"1.14"},{"name":"Odh-Elyra","version":"4.0"},{"name":"PyMongo","version":"4.8"},{"name":"Pyodbc","version":"5.1"}, {"name":"Codeflare-SDK","version":"0.21"}, {"name":"Sklearn-onnx","version":"1.17"}, {"name":"Psycopg","version":"3.2"}, {"name":"MySQL Connector/Python","version":"9.0"}]'
```

Comment:
```suggestion
        opendatahub.io/notebook-python-dependencies: '[{"name":"TensorFlow","version":"2.17"},{"name":"Tensorboard","version":"2.17"},{"name":"Nvidia-CUDA-CU12-Bundle","version":"2.13"},{"name":"Boto3","version":"1.35"},{"name":"Kafka-Python-ng","version":"2.2"},{"name":"Kfp","version":"2.5"},{"name":"Matplotlib","version":"3.9"},{"name":"Numpy","version":"1.26"},{"name":"Pandas","version":"2.2"},{"name":"Scikit-learn","version":"1.5"},{"name":"Scipy","version":"1.14"},{"name":"Odh-Elyra","version":"4.0"},{"name":"PyMongo","version":"4.8"},{"name":"Pyodbc","version":"5.1"}, {"name":"Codeflare-SDK","version":"0.21"}, {"name":"Sklearn-onnx","version":"1.17"}, {"name":"Psycopg","version":"3.2"}, {"name":"MySQL Connector/Python","version":"9.0"}]'
```

Reply:
Added as well a new annotation for the embeded cuda that comes along with tensorflow called `{"name":"Nvidia-CUDA-CU12-Bundle","version":"2.13"}`

Reply:
I guess that the expected version in this discussion was `12.3` and not `2.13`:
```
(app-root) pip list | grep cuda
nvidia-cuda-cupti-cu12       12.3.101
nvidia-cuda-nvcc-cu12        12.3.107
nvidia-cuda-nvrtc-cu12       12.3.107
nvidia-cuda-runtime-cu12     12.3.101
```
I raised the #742.